frontend/src/service/api-service.js

```diff
@@ -0,0 +1,7 @@
+import axios from "axios";
+
+export const getPlayer = () =>
+    axios.get("/").then(response => response.data)
+
+export const postPlayer = (name) =>
+    axios.post(`api/HighscoreDarts/dart/${name}`, "string")
```

Comment:
```suggestion
export const postPlayer = (name) =>
    axios.post(`api/HighscoreDarts/dart`,  {name})
```

See comment in the controller